airbyte-integrations/connectors/destination-bigquery/src/main/kotlin/io/airbyte/integrations/destination/bigquery/spec/BigquerySpecification.kt

```diff
@@ -31,8 +31,6 @@ import java.io.IOException
 
 @Singleton
 class BigquerySpecification : ConfigurationSpecification() {
-    val numOpenStreamWorkers: Int? = null
```

Comment:
this was just for easier testing. Internal workspace behavior says we should just hardcode to 3.